packages/perseus/src/widgets/__tests__/unit_test.jsx

```diff
@@ -1,144 +1,166 @@
 // @flow
+import "@testing-library/jest-dom";
+import {render, screen} from "@testing-library/react";
+import React from "react";
+
 import Unit, {countSigfigs, sigfigPrint} from "../unit.jsx";
 
 const {widget} = Unit;
 const UnitWidget = widget;
 
-describe("countSigfigs", () => {
-    it("gets a few simple cases right", () => {
-        expect(countSigfigs("5")).toEqual(1);
-        expect(countSigfigs("500")).toEqual(1);
-        expect(countSigfigs("5.00")).toEqual(3);
-        expect(countSigfigs("5.01")).toEqual(3);
-        expect(countSigfigs("05.01")).toEqual(3);
-    });
-});
+describe("Unit Widget", () => {
+    it("renders", () => {
+        const props = {
+            onBlur: () => {},
+            onFocus: () => {},
+            onChange: () => {},
+        };
 
-describe("displaySigFigs", () => {
-    it("gets a few simple cases right", () => {
-        expect(sigfigPrint(150, 3)).toEqual("150.");
-        expect(sigfigPrint(0.0005, 2)).toEqual("5.0 x 10^-4");
-        expect(sigfigPrint(15000, 2)).toEqual("1.5 x 10^4");
-        expect(sigfigPrint(50, 2)).toEqual("50.");
+        render(<Unit.widget {...props} />);
+
+        expect(screen.getByTestId("perseus__unit-input")).toBeInTheDocument();
```

Comment:
**suggestion (non-blocking):** I'd probably use an inline snapshot for this instead of a test ID given how small the markup is for this component.